### DISCUSSION TITLE
Cannot duplicate screen in version v1.0.145

### DISCUSSION THREAD

**Discussion by xnohat** [Budibase/budibase#5740] [General]
Hi all,

I updated to new version v1.0.145 and got an issue when Duplicate Screens, got this error :( 

![image](https://user-images.githubusercontent.com/4613808/167070260-080acd0b-e740-49c9-8c29-18003e113696.png)

any maintainer can confirm this issue and where is parts of code can make this, I will try to contribute a fix to it 

Thanks

**Comment by mjashanks** [comment]
Hey @xnohat - not a code maintainer, but I believe this is it: https://github.com/Budibase/budibase/blob/develop/packages/builder/src/components/design/NavigationPanel/ComponentNavigationTree/ScreenDropdownMenu.svelte

Thank you :)

**Reply by xnohat** [reply]
Thanks for your suggest, I have fixed this issue.
https://github.com/Budibase/budibase/pull/5758
